pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -140,11 +129,9 @@ stdenv.mkDerivation (
       libxml2
     ];
 
-    env = lib.optionalAttrs ((lib.versionAtLeast "28.0-rc1" version) && ex_docSupport) {
-      # erlang-28.0-rc> warning: jinterface.html redirects to ../lib/jinterface/doc/html/index.html, which does not exist
-      # erlang-28.0-rc>
-      # erlang-28.0-rc> warning: odbc.html redirects to ../lib/odbc/doc/html/index.html, which does not exist
-      EX_DOC_WARNINGS_AS_ERRORS = "false";
+    env = {
+      # only build shell/IDE docs and man pages
+      DOC_TARGETS = "chunks man";
```

Comment:
Just an FYI that in the future the `man` target might use ExDoc, so you may have to re-introduce the ex_doc support then.

Reply:
Thanks for the heads up. I may argue for dropping the man pages completely at that point, rather than going back to building erlang twice. I suspect most uses of erlang from Nix are packages and development shells, not host-level installations.